mtags-shared/src/main/scala/scala/meta/internal/mtags/CoursierComplete.scala

```diff
@@ -20,35 +20,70 @@ class CoursierComplete(scalaVersion: String) {
       else scalaVersion.split('.').take(2).mkString(".")
     )
 
+  private def completions(s: String): List[String] = {
+    val futureCompletions = Future {
+      api.withInput(s).complete().getCompletions().asScala.toList
+    }
+    try Await.result(futureCompletions, 10.seconds)
+    catch {
+      case _: Throwable => Nil
+    }
+  }
+
   def complete(
       dependency: String,
-      includeScala: Boolean = true
+      includeScala: Boolean = true,
+      supportNonJvm: Boolean = true
   ): List[String] = {
+    // Version completions
+    if (dependency.replaceAll(":+", ":").count(_ == ':') == 2) {
+      versionCompletions(dependency, supportNonJvm)
+    } else {
+      val javaCompletions = completions(dependency)
+      val scalaCompletions =
+        if (
+          includeScala &&
+          dependency.endsWith(":") && dependency.count(_ == ':') == 1
+        )
+          completions(dependency + ":").map(":" + _)
+        else List.empty
 
-    def completions(s: String): List[String] = {
-      val futureCompletions = Future {
-        api.withInput(s).complete().getCompletions().asScala.toList
-      }
-      try Await.result(futureCompletions, 10.seconds)
-      catch {
-        case _: Throwable => Nil
-      }
+      (scalaCompletions ++ javaCompletions).distinct
     }
+  }
 
-    val javaCompletions = completions(dependency)
-    val scalaCompletions =
-      if (
-        includeScala &&
-        dependency.endsWith(":") && dependency.count(_ == ':') == 1
-      )
-        completions(dependency + ":").map(":" + _)
-      else List.empty
+  private def versionCompletions(
+      dependency: String,
+      supportNonJvm: Boolean
+  ): List[String] = {
+    val (adjusted, hasDoubleColon) = adjustDoubleColon(dependency)
+    val sortedCompletions = completions(adjusted).sortWith(
+      Version.fromString(_) >= Version.fromString(_)
+    )
+    if (!hasDoubleColon && supportNonJvm)
+      sortedCompletions.map(":" + _)
```

Comment:
Actually, we shouldn't add `:` in version completions but in module name completions, no? So if name ends with `sjs1` or `native0.4` etc, we can strip it and add `::`

Reply:
I think it can be confusing for users, that they choose name ending with `sjs1` and we remove it?

Reply:
Hmm... you might be right :thinking: Though I think on the other hand users might never discover that functionality. We would need to filter out native libraries and add :: automatically if those exist, but that could be a flaky.

Reply:
So anyway, it's fine to not add it.